test/slider-test.jsx

```diff
@@ -41,11 +41,8 @@ describeWithDOM('<Slider />', () => {
 
     it('renders pits if they are provided', () => {
       const pitRender = sinon.stub().returns(<div />);
-
-      // eslint-disable-next-line
-      const PitComponent = React.createClass({
```

Comment:
We need to continue to test `React.createClass`; please use the `create-react-class` package as a dev dep instead.